server/lib/knex.js

```diff
@@ -17,6 +17,15 @@ const knex = require('knex')({
             'DATETIME'
         ]
     },
+    pool: {
+        min: 2,
+        max: 10,
+        afterCreate: function(conn, cb) {
+            conn.query('SET sql_mode="ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION";', function (err) {
```

Comment:
What is the reason for this? Could we do it without? I don't wont to include stuff which does not address any particular problem. Thanks.

Reply:
Since MySQL 5.7 STRICT_TRANS_TABLES is enabled by default, you can disable globally in my.cnf or in database connection. For docker in this issue https://github.com/Mailtrain-org/mailtrain/issues/748 fix removing all sql_modes. I prefer remove only STRICT_TRANS_TABLES and keep the others default values ERROR_FOR_DIVISION_BY_ZERO, NO_AUTO_CREATE_USER, NO_ENGINE_SUBSTITUTION. I thing that adding in afterCreate function of knex connection is the better place to solve it.

Reply:
With STRICT_TRANS_TABLES enabled, mailtrain do not save any field of type date or timestamp, throwing ER_TRUNCATED_WRONG_VALUE

Reply:
But why is it? As far as I understand, It disables checking, but we should not fill in invalid data anyway.

Reply:
I think that if all inserts and updates will be called with 'YYYY-MM-DD HH:mm:ss' format in UTC should works without database errors, but i did not find where are in source code. This workaround is better that reconfigure manually the database for date fields begin to works and it can be easily remove if we find another solution.

Reply:
I think we have to find the problem. In my view, disabling the check just masks the problem.

Reply:
![image](https://user-images.githubusercontent.com/1895563/92615854-dab11300-f2bd-11ea-84ae-972ec066c9cd.png)
Maybe change .toISOString() in subscriptions models to .format("YYYY-MM-DD HH:mm:ss") can be solve it...

Try?

Reply:
Seems to works! :) In a while push the fix to this PR